src/database/models/bus.js

```diff
@@ -15,20 +15,21 @@ module.exports = (sequelize, DataTypes) => {
         foreignKey: 'userId',
         as: 'driver',
         onDelete: 'CASCADE',
-      })
+      });
     }
-  };
+  }
+
   Bus.init({
     busPlate: DataTypes.STRING,
     busStatus: DataTypes.STRING,
     busLocation: DataTypes.STRING,
     busCompany: DataTypes.STRING,
     busSeats: DataTypes.STRING,
-    userId: DataTypes.STRING
+    userId: DataTypes.STRING,
+    routeId: DataTypes.INTEGER,
```

Comment:
Does a bus has to be always associated with a route?

Reply:
yeah, I think so cause I think no buses can exist without having a route which it will be using in its activities.